src/tdycore.c

```diff
@@ -464,11 +478,13 @@ PetscErrorCode TDyView(TDy tdy,PetscViewer viewer) {
   PetscFunctionReturn(0);
 }
 
-/// Sets options for the dycore based on command line arguments supplied by a
-/// user. TDySetFromOptions must be called before TDySetup,
-/// since the latter uses options specified by the former.
-/// @param [inout] tdy The dycore instance
-PetscErrorCode TDySetFromOptions(TDy tdy) {
+/// Reads command line options specified by the user
+///
+/// @param [inout] tdy A TDy struct
+///
+/// @returns 0 on success, or a non-zero error code on failure
+PetscErrorCode ReadCommandLineOptions(TDy tdy) {
```

Comment:
If you want to make this an "internal" function visible only within this file, you can add the `static` keyword before the function's declaration.

Reply:
Fixed.